### PR TITLE
Fix modexp checks

### PR DESCRIPTION
**Pull request by laisolizq**

### CODE REVIEW

File: core/vm/contracts_zkevm.go [hunk 1]
```diff
@@ -301,29 +301,6 @@ func (c *bigModExp_zkevm) RequiredGas(input []byte) uint64 {
 	}
 
 	// Retrieve the operands and execute the exponentiation
```
Comment (hexoscott) [inline core/vm/contracts_zkevm.go]: Is it worth removing this comment as well now the code block is gone?

**Review by hexoscott — CHANGES_REQUESTED** [review]
One tiny thing around removing a comment.

Looking at the code there isn't a change in handling some new case, rather just shuffling code to handle a new edge case?  Just checking that we don't need to add any more unit tests as part of this, seems that we don't.

**Review by hexoscott — APPROVED** [review]
